lib/jobs/node-discovered-alert-job.js

```diff
@@ -43,16 +45,22 @@ function nodeAlertJobFactory(
      */
     NodeAlertJob.prototype._run = function run() {
         var self = this;
+        var nodeIp = lookupService.nodeIdToIpAddresses(self.nodeId);
```

Comment:
the return value is a promise. it's easily confused, if it's not in a promise chain